src/Bridges/Constraint/bridges/split_complex_zeros.jl

```diff
@@ -34,6 +34,7 @@ struct SplitComplexZerosBridge{
     G<:MOI.Utilities.TypedLike{Complex{T}},
 } <: AbstractBridge
     dimension::Int
+    func::G  # Simplify querying ConstraintFunction.
```

Comment:
We don't want to simplify it. It's ok for it to be slow. However, we want bridges to have low memory footprint

Reply:
I've been prioritizing correctness and completeness over performance. But yeah, we can fix this in a different PR if you like.

Reply:
Yes, thanks for making the bridges tests working with complex! Just wanted to check we're on the same page about this which is why I also clarified it in the docs now